tests/core/structure/sheet.test.ts

```diff
@@ -20,4 +21,98 @@ describe("Sheet", () => {
     expect(cell).toBeInstanceOf(Cell);
     expect(cell!.formula).toBe(value);
   });
+
+  it("should create cells with references which are reflected by cell.referencesIn and cell.referencesOut", () => {
+    const setCell = function (col: number, row: number, value: string) {
+      const cInfo = sheet.setCellAt(col, row, value);
+      const cKey = sheet.columns
+        .get(cInfo.position.columnKey!)!
+        .cellIndex.get(cInfo.position.rowKey!)!;
+      return sheet.cellData.get(cKey)!;
+    };
```

Comment:
Why is this function needed? It seems like a proxy function with a new return data but the returned data were never used

Reply:
The return data is used to populate the cells array below it, which is used to read the references of the Cell objects.

Reply:
I would suggest this change, because it is not obvious what is going in here


```suggestion
    const getCellReference = (cInfo: CellInfo) => {
      const cKey = sheet.columns
        .get(cInfo.position.columnKey!)!
        .cellIndex.get(cInfo.position.rowKey!)!;
      return sheet.cellData.get(cKey)!;
    };

    const cells = [
      getCellReference(sheet.setCellAt(0, 0, "100")), // A1
      getCellReference(sheet.setCellAt(1, 0, "=A1")), // B1
      getCellReference(sheet.setCellAt(2, 2, "1")), // C3
      getCellReference(sheet.setCellAt(2, 0, "=A1 + B1 + C3")), // C1
    ];
```